core-ui/src/components/Predefined/Details/Pod/Pod.details.js

```diff
@@ -29,6 +30,14 @@ function goToSecretDetails(resourceKind, name) {
 
 export const PodsDetails = ({ DefaultRenderer, ...otherParams }) => {
   const { t, i18n } = useTranslation();
+
+  const filterByHost = e =>
+    e.involvedObject.name === otherParams.resourceName &&
+    e.involvedObject.kind === 'Pod';
```

Comment:
We have this function used many times, the only difference is the kind used. Can we move it to shared and import it, instead of writing the same code multiple times?

Reply:
Also, I would add `?` just in case the `involvedObject` is undefined - we know that k8s likes to surprise us this way from time to time